src/OrchardCore/OrchardCore.DisplayManagement/OrchardCoreBuilderExtensions.cs

```diff
@@ -98,7 +98,11 @@ public static OrchardCoreBuilder AddTheming(this OrchardCoreBuilder builder)
                     services.AddTagHelpers<ShapeTagHelper>();
                     services.AddTagHelpers<ValidationMessageTagHelper>();
                     services.AddTagHelpers<ZoneTagHelper>();
-                });
+                })
+                .ConfigureServices(services => {
+                    // Configure default services
+                    services.AddScoped<IShapePlacementProvider, ShapeTablePlacementProvider>();
+                }, order: -10);
```

Comment:
Yes i talked about using the `ConfigureServices()` helper if we would want to register it last

But because things are combined we want to register it before

For info, for a given order value, e.g. the default value of 0, it will be executed before the `ConfigureServices()` of any module startup having the same order value. **So maybe we don't need it**, this is only if someone want to register another `IShapePlacementProvider` from the app, that he would have to use a positive order.